governance/what-is-governance.md

```diff
@@ -0,0 +1,69 @@
+# What is Governance?
+## Governance of Open Source Projects
+The history of the internet is rooted in open source. 
+
+‘Open source’ refers to software projects which are [freely open and available to all](https://www.theopensourceway.org/the_open_source_way-guidebook-2.0.html).  They are developed and maintained according to the principles of transparency, collaboration, accessibility (“release early and often”), inclusive meritocracy, and community. 
+
+Open source code can be used, copied, changed, and redistributed by anyone.  Anyone can likewise propose changes or additions to the original project, though these proposals are not always accepted. 
+
+Many individuals and organizations contribute to the maintenance and development of open source projects- such as [TensorFlow](https://www.tensorflow.org/), [React](https://react.dev/), and [Ethereum](https://ethereum.org/en/)- though no one person or entity “owns” or controls the entire project. 
```

Comment:
s#Many individuals and organizations contribute to the maintenance and development of open source projects- such as [TensorFlow](https://www.tensorflow.org/), [React](https://react.dev/), and [Ethereum](https://ethereum.org/en/)#
Many individuals and organizations contribute to the maintenance and development of open source projects- such as [Linux](https://www.kernel.org/), [Debian](https://debian.org), and [Ethereum](https://ethereum.org/en/)-#

(So Tensorflow and React aren't perfect examples for the language that follows this, because they are both managed by companies -- Google and Facebook; and require [Contributor License Agreements](https://cla.developers.google.com/about/google-individual?csw=1) which pass ownership of contributions to these companies. Sugget Debian and Linux because they're closer to the ideal described).